gateleen-hook-js/src/test/javascript/hook/gateleen-hook.test.js

```diff
@@ -72,25 +72,6 @@ describe('hook', function(){
       $httpBackend.verifyNoOutstandingExpectation();
       $httpBackend.verifyNoOutstandingRequest();
     });
-    it('should create the hook with HTTP PUT from complete URL', function(){
-      $httpBackend.expectPUT('http://server/context/path/_hooks/listeners/http/gateleen-hook-js-0123',
```

Comment:
The current production code doesn't support this case and it doesn't seem to be in use/required. The tests were not executed on Travis, hence we didn't notice.